.github/workflows/release.yml

```diff
@@ -413,6 +413,47 @@ jobs:
           name: logs-warp-sync
           path: logs
 
+  chopsticks-upgrade-test:
+    runs-on:
+      labels: ubuntu-latest
+    needs: ["set-tags", "build"]
+    strategy:
+      matrix:
+        chains: [
+          { chain: "stagenet_dancebox", runtime: "dancebox" },
+          { chain: "dancebox", runtime: "dancebox" },
+        ]
+    env:
+      GH_WORKFLOW_MATRIX_CHAIN: ${{ matrix.chains.chain }}
+      DEBUG_COLORS: 1
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ needs.set-tags.outputs.git_ref }}
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 20.x
+          cache: "pnpm"
+          cache-dependency-path: test/pnpm-lock.yaml
```

Comment:
with workspaces (which is what tanssi uses) you dont need to add the direct path